src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInNameCheck.java

```diff
@@ -190,6 +217,9 @@
     /** Allow to skip variables with {@code static} modifier. */
     private boolean ignoreStatic = true;
 
+    /** Allow to skip variables with both {@code static} and {@code final} modifiers. */
```

Comment:
we need to explain users how this property value will affect values ignoreStatic and ignoreFinal.
this property in value `false` make conflicts with previous behavior and it affects other properties values (other properties are ignored if this property found match in code)

Reply:
I added a paragraph to the description in xdoc to explain the behavior of `ignoreStaticFinal`.